testing/mock/ack.go

```diff
@@ -0,0 +1,23 @@
+package mock
+
+// MockEmptyAcknowledgement implements the exported.Acknowledgement interface and always returns an empty byte string as Response
+type MockEmptyAcknowledgement struct {
```

Comment:
@colin-axner Open to suggestions with this. I don't want to add technical debt unnecessarily.

Reply:
No this is great. None of the mock code has added any technical debt so far. I think it'll be useful to keep around